### PR TITLE
Add-remove Tasks

### PR DESCRIPTION
**Pull request by Ofarouq310**
## Summary of branch additions: 
- [x] Implement a function for adding and deleting a task.
- [x] Implement a function for editing task descriptions.
- [x] "Completed" property is set to false by default.
- [x] Updating tasks indexes upon deleting tasks.

### CODE REVIEW

File: src/modules/addDeleteTask.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+           </ul>
+         </div>
+         <img src="./images/dots.png" alt="dots" class="dots_icon" id="${task.id}"</img>  
+    </li>`;
+
+  list.innerHTML += temp;
+
+  threeDots();
+
+  editTask();
```
Comment (gemmen29) [inline src/modules/addDeleteTask.js:38]: - [x] Kindly fix the `edit` task functionality by updating the local storage when you edit any task as now when I press the edit button then enter a new value for a task and try to refresh the page the task remains the same as the old state.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Ofarouq310, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @Ofarouq310, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Red-approved

## Good Highlights

- Linter checks are passing
- Adherence to GitHub flow
- Repo is professional
- Good commit messages

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rloterh — COMMENTED** [review]
## Status: Approved :heavy_check_mark:

Hi Omar,  

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: . Congratulations!  🎉🌟 

## Highlights 👍
- Good job

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
- I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor. 
- MIT license link in your ReadMe is broken, kindly endeavor to fix it.
Implementing the above change(s) will make your ReadMe/ repository stand out even better 😉

keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**